docs/how-to-upload-manifest.md

```diff
@@ -0,0 +1,120 @@
+# How To Push Operator Manifest To App Registry
+This document shows how one can push operator manifest(s) into `quay.io` or any other compatible app registry server.
+
+```
```

Comment:
Why are you using ``` here? This should be a regular paragraph. If you want to call out something, please use bold.

Reply:
It's like an info section, looks more aesthetic in my opinion :). I would reserve making an entire line bold for something really really critical.

Reply:
What is aesthetic is a matter of personal choice. I would rather follow the general standard of using ``` ``` to indicate code and stuff that shouldn't be formatted. I definitely want this standard to be followed in our docs.